test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -71,7 +112,23 @@ public static void dropAllObjects() throws SQLException {
         throw new AssertionError(e);
       }
     }
-    executeUpdate("drop all objects");
+    if (DATABASE.equalsIgnoreCase("h2")) {
+      executeUpdate("drop all objects");
+    } else if (DATABASE.equalsIgnoreCase("sqlserver")) {
+      dropSQLServerTables();
+      // TODO (srinivas): drop tables for Oracle database.
+    }
+  }
+
+  private static void dropSQLServerTables() throws SQLException {
+    String sql = "select name FROM sys.tables";
```

Comment:
I find this reads much better if you just drop most of the variables (sql, name, dropQuery). It may shrink enough, even with the try-with-resources mentioned elsewhere, to inline this method. Also, we don't generally uppercase SQL keywords, but here you're mixing upper- and lowercase in one string. I'd just stick to lowercase, here and for "drop table".